components/automate-cs-ocid/habitat/plan.sh

```diff
@@ -62,10 +70,6 @@ do_install() {
   echo "gem 'tzinfo-data'" >> Gemfile
```

Comment:
Add a comment that we need to remove this one, once we have oc-id upgrade done.

Reply:
sure

Reply:
There is already a TODO note left above this line.